app/controllers/questions_controller.rb

```diff
@@ -0,0 +1,46 @@
+class QuestionsController < ApplicationController
+  def index
+    @questions = Question.all
+  end    
+
+  def show
+    @question = Question.find(params[:id])
+    
+    if @question.resolved 
+      @resolution_status = "Resolved"
+    else
+      @resolution_status = "Not resolved"
+    end
+  end
+
+  def new
+    @question = Question.new
+  end
+
+  def edit
+     @question = Question.find(params[:id])
+  end
+
+  def create
+    @question = Question.new(params.require(:question).permit(:title, :body))
+    if @question.save
+      flash[:notice] = "Question was saved."
+      redirect_to @question
+    else
+      flash[:error] = "Please save question again."      
+      render :new
+    end
+  end
+  
+  def update
+    @question=Question.find(params[:id])
+    if @question.update_attributes(params.require(:question).permit(:title, :body, :resolved))
```

Comment:
The `params` section of this line is very similar to line 25 in `#create`.  Could you move this logic to a method to make a more DRY?